style.css

```diff
@@ -1 +1,319 @@
 /*  Add your styling here */
+@import url('https://fonts.googleapis.com/css2?family=Happy+Monkey&family=Montserrat&family=Nosifer&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-size: 16px;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(3, 1fr);
+    grid-column-gap: 0px;
+    grid-row-gap: 0px;
+}
+
+/* -----header------ */
+
+header {
+    grid-area: 1 / 1 / 2 / 2;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    grid-column-gap: 0px;
+    grid-row-gap: 0px;
+    background-color: #61d4af;
+}
+
+.logo {
+    grid-area: 1 / 1 / 3 / 2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 150px;
+}
+.logo img {
+    width: 300px;
+    height: 300px;
+    
+}
+
+nav {
+    grid-area: 1 / 2 / 2 / 3;
+}
+
+.menu {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
```

Comment:
Did the menu need to be an absolute position here? You have set up the main page to use a grid system, so it would be best to continue in this style throughout the exercise. There's a good article on using a full grid to layout a page here: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/